packages/components/src/components/badge/badge.stories.ts

```diff
@@ -26,110 +36,51 @@ export default {
  */
 
 export const Default = {
+  name: 'Default',
   render: (args: any) => {
     return generateTemplate({ args });
   }
 };
 
 /**
- * The badge in all possible combinations of `variant` and `size`.
+ * Use the `size` attribute to set the badge’s size. The default size is `md` but you can also use `sm` and `lg`.
  */
 
-export const VariantAndSize = {
-  name: 'Variant × Size',
-  parameters: { controls: { exclude: ['variant', 'size'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'variant' },
-        y: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
+export const Size = {
+  name: 'Size',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-badge size="sm">8</sd-badge>
+      <sd-badge size="md">8</sd-badge>
+      <sd-badge size="lg">8</sd-badge>
+    </div>
+  `
 };
 
 /**
- * The badge in all possible combinations of `variant` and `inverted`.
+ * Use the `variant` attribute to set the badge’s variant. The default variant is `default` but you can also use `success` and `error`.
```

Comment:
```suggestion
 * Use the `variant` attribute to set the badge’s variant.
```

Less is more.

Reply:
Adapted in a similar way to the sd-button